thin-client/src/thin_client.rs

```diff
@@ -111,6 +111,7 @@ impl ClientOptimizer {
 }
 
 /// An object for querying and sending transactions to the network.
+#[deprecated(since = "2.0.0", note = "Use [RpcClient] or [TpuClient] instead.")]
```

Comment:
```suggestion
#[deprecated(since = "1.19.0", note = "Use [RpcClient] or [TpuClient] instead.")]
```
Since this is the repo's current version, let's go with this for now. I'll mention it to releng, but hopefully we can all remember to update these to v2 when that version bump lands.